### DISCUSSION TITLE
Display firmware Added LCD backlight brightness control

### DISCUSSION THREAD

**Discussion by RUtarah** [Jyers/Marlin#384] [Q&A] [answered]
Hi, I just saw this update and try to flash my screen. But no any change for brightness control. Am I doing it wrong? Screen already change from blue to orange. Thanks

**Comment by joeclifford** [comment, marked as answer]
You don't need to flash the screens firmware for this to work.

This update adds a new setting in the Advanced menu to allow you change the screen brightness.

The reason you may want to flash the screen firmware is to get the new icons for the brightness menu, but it isn't needed.

**Reply by RUtarah** [reply]
Thanks, I thought this function available when flash screen firmware. But after compile and flash mboard firmware I find this new icon. Thanks ya!
